akka-testkit/src/main/scala/akka/testkit/TestKit.scala

```diff
@@ -688,6 +724,48 @@ trait TestKitBase {
     TestKit.shutdownActorSystem(actorSystem, duration, verifySystemShutdown)
   }
 
+  // actorOf is defined without defaults for Java API compatibility
+  /**
+   * Spawns an actor as a child of this test actor, and returns the child's ActorRef.
+   * The child will stop on failure.
+   */
+  def actorOf(props: Props): ActorRef = {
```

Comment:
Since this only creates children of the test actor, and since the term "test actor" is sometimes hidden (when using plain `TestProbe`), how about just `createChild` or `childActorOf`?

I personally prefer to stick closer to akka's own terminology, hence including "actor of" in the name.